runtime/rasdump/trigger.c

```diff
@@ -604,14 +604,13 @@ prepareForDump(struct J9JavaVM *vm, struct J9RASdumpAgent *agent, struct J9RASdu
 
 		/* Share exclusive access when it's a "slow entry" or "user" event, as there may be a deadlock */
 		UDATA shareVMAccess = exclusiveHeld &&
-				( (context->eventFlags & J9RAS_DUMP_ON_USER_SIGNAL) ||
-				(context->eventFlags & J9RAS_DUMP_ON_SLOW_EXCLUSIVE_ENTER) );
+				OMR_ARE_ANY_BITS_SET(context->eventFlags, J9RAS_DUMP_ON_USER_SIGNAL | J9RAS_DUMP_ON_SLOW_EXCLUSIVE_ENTER);
```

Comment:
Point noted about the `&&` but the idea was to avoid including `J9RAS_DUMP_ON_USER2_SIGNAL` in this condition to not share exclusive access if held by someone else to resolve the use case in https://github.com/eclipse-openj9/openj9/issues/9256